daemon/archive.go

```diff
@@ -375,7 +375,7 @@ func (daemon *Daemon) CopyOnBuild(cID, destPath, srcRoot, srcPath string, decomp
 
 	destExists := true
 	destDir := false
-	rootUID, rootGID := daemon.GetRemappedUIDGID()
+	rootIDs, _ := daemon.idMappings.RootPair()
```

Comment:
Mainly just because that's what the old code did: https://github.com/moby/moby/pull/33362/files#diff-1a1f3e7ad9b1d7584e2d3e7d0c4c3db9L969

I'm not sure what error is possible here. I'll take another look. Maybe it doesn't make sense to return an error at all.